ci/do_ci.sh

```diff
@@ -180,8 +180,10 @@ elif [[ "$1" == "fix_format" ]]; then
   ./tools/check_format.py fix
   exit 0
 elif [[ "$1" == "check_format" ]]; then
-  echo "check_format..."
   cd "${ENVOY_SRCDIR}"
+  echo "check_format_test..."
```

Comment:
Should this be an independent test run by CI rather than part of `check_format`? I'd suggest we don't force Docker on folks who are using `check_format` natively, as not everyone is able to run Docker in their environment.

Reply:
2 separate questions I think.

1. should it be an independent test run by CI?  It could, but it's extremely quick and not noisy by default. The overhead for setting up a new test is significantly higher.

2. Actually check_format_test_run_under_docker.py does not invoke docker. It just expects to be run with the correct tools in the toolchain. I wanted to dissuade people from running the Python script directly and get all kinds of false errors if the wrong clang is installed. But if you are able to run check_format.py and get compatible results, then check_format_test_run_under_docker.py will also work.

Do you have suggestions about how to convey that more clearly?

Reply:
I would drop the `_under_docker` suffix, since we don't use this for `check_format.py` itself. If they have shared fate, they should use the same convention I reckon.

Reply:
Done; not sure what's happening with CI; got local error:

fatal: unable to access 'https://github.com/envoyproxy/envoy-filter-example.git/': Could not resolve host: github.com

which I thought might be related to MSFT? Still, pushing commit to github seemed to work.